### PR TITLE
fix(COR-966): external_host field doesn’t work for Helm

### PR DESCRIPTION
**Pull request by pggb25**

### CODE REVIEW

File: internal/domain/helm/helm.go [hunk 1]
```diff
@@ -1,10 +1,13 @@
 package helm
 
 import (
+	"fmt"
+	"github.com/AlekSi/pointer"
 	"github.com/go-playground/validator/v10"
 	"github.com/google/uuid"
 	"github.com/pkg/errors"
 	"github.com/qovery/qovery-client-go"
+	"strings"
```
Comment (ctjhoa) [inline internal/domain/helm/helm.go:10]:
```suggestion
	"fmt"
	"strings"
		
	"github.com/AlekSi/pointer"
	"github.com/go-playground/validator/v10"
	"github.com/google/uuid"
	"github.com/pkg/errors"
	"github.com/qovery/qovery-client-go"
```

Comment (ctjhoa) [reply internal/domain/helm/helm.go:10]: Due to linting rule

File: internal/domain/helm/helm.go [hunk 2]
```diff
@@ -229,6 +233,10 @@ func (h *Helm) SetEnvironmentVariables(vars variable.Variables) error {
 	h.EnvironmentVariables = envVars
 	h.BuiltInEnvironmentVariables = builtIn
 
+	if err := h.SetHosts(vars); err != nil {
+		return nil
```
Comment (ctjhoa) [inline internal/domain/helm/helm.go:237]:
```suggestion
		return err
```

**Review by ctjhoa — CHANGES_REQUESTED** [review]

File: internal/domain/helm/helm.go (hunk 2 above)
Comment (pggb25) [reply internal/domain/helm/helm.go]: done

File: internal/domain/helm/helm.go [hunk 3]
```diff
@@ -263,3 +271,28 @@ func (h *Helm) SetState(st status.State) error {
...
+			h.InternalHost = pointer.ToString(v.Value)
+			continue
+		}
+		if h.ExternalHost != nil && h.InternalHost != nil {
+			return nil
+		}
+	}
+
+	return ErrFailedToSetHosts
+}
```
Comment (mzottola) [inline internal/domain/helm/helm.go:298]: Looks like the this method doesn't return any error. 
The linter will complain iirc @ctjhoa ?

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**